apps/src/templates/studioHomepages/CourseBlocksGradeBands.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import ContentContainer from '../ContentContainer';
+import ResourceCard from './ResourceCard';
+import styleConstants from '../../styleConstants';
+import i18n from "@cdo/locale";
+
+const pegasusContentWidth = styleConstants['content-width'];
```

Comment:
here and elsewhere: should we just be calling this `contentWidth` rather than `pegasusContentWidth`?